x-pack/plugins/ml/public/application/model_management/add_model_flyout.tsx

```diff
@@ -187,6 +197,38 @@ const ElserTabContent: FC<ElserTabContentProps> = ({ modelDownloads, onModelDown
               </div>
             ) : null}
 
+            {modelName === 'e5' ? (
+              <div>
+                <EuiTitle size={'s'}>
+                  <h3>
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.modelsList.e5Title"
+                      defaultMessage="E5 (EmbEddings from bidirEctional Encoder rEpresentations)"
+                    />
+                  </h3>
+                </EuiTitle>
+                <EuiSpacer size="s" />
+                <p>
+                  <EuiText color={'subdued'} size={'s'}>
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.addModelFlyout.e5Description"
+                      defaultMessage="E5 is an NLP model that enables you to perform multi-lingual semantic search by using dense vector representations. This model is recommended for non-English language documents and queries."
+                    />
+                  </EuiText>
+                </p>
+                <EuiSpacer size="s" />
+                <p>
+                  <EuiLink href={docLinks.links.ml.nlpElser} external>
```

Comment:
Does `external` not open inside a separate tab when used in a flyout? Ideally I think the docs links should open in a new tab.

Reply:
Also do we have a separate link for the E5 model? Currently it is opening the ELSER docs page.

Reply:
yes, I'm waiting for an update from the docs team